src/main/java/com/gooddata/executeafm/resultspec/AttributeSortItem.java

```diff
@@ -45,6 +58,10 @@ public String getAttributeIdentifier() {
         return attributeIdentifier;
     }
 
+    public String getAggregation() {
```

Comment:
would be nice to add javadoc explaining the semantics, since "aggregation" by this object is not very intuitive (at least to me)

Reply:
https://confluence.intgdc.com/display/analysis/Sorting+in+StackBar+Chart+-+API+Changes

Reply:
Added detailed description to javadoc of AttributeSortItem.